src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/csip/CSIPResource.kt

```diff
@@ -386,6 +389,16 @@ data class Attendee(
   val contribution: String? = null,
 )
 
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
CSIP documents are associated with the booking - but since all csip reports have a booking id - I thought it would be useful to return document summary data